libqfieldsync/offliners.py

```diff
@@ -53,6 +53,7 @@ def convert_to_offline(
         offline_db_filename: str,
         layers: List[QgsMapLayer],
         bbox: Optional[QgsRectangle],
+        exported_project_title: Optional[str] = "",
```

Comment:
```suggestion
        exported_project_title: str = "",
```

This is not optional anymore.